src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -211,6 +216,15 @@ private void validateSchema(final EventTypeBase eventType) throws InvalidEventTy
         }
     }
 
+    @VisibleForTesting
```

Comment:
For me this annotation always smells, because it means that you want to test something that is supposed to be private. I think a better solution can be to create a separate class that does only json validation and put this method there. And EventTypeService will use this class.